pyanaconda/geoloc.py

```diff
@@ -192,8 +193,9 @@ def _check_if_geolocation_should_be_used(self, options_override):
                 geolocation_enabled = False
 
         # and also check if geolocation was not disabled by boot or command like option
-        if not flags.cmdline.getbool('geoloc', True):
-            # flags.cmdline.getbool is used as it handles values such as
+        if not kernel_arguments.get('geoloc', default=True, interpret_booleans=True) or \
+                "nogeoloc" in kernel_arguments:
```

Comment:
Actually, the option `nogeoloc` is not documented either. It is recommended to use  `inst.geoloc=0` to disable a geolocation.

Reply:
Agree, let's remove this. If it is not documented we can use that as argument for removal.

Reply:
Removed.